### PR TITLE
SONiC quality guarantee definition

### PR DESCRIPTION
**Pull request by liat-grozovik**
Quality guarantee definition following TSC discussion and recommendation.

### CODE REVIEW

**Comment by zhangyanzhao** [comment]
@liat-grozovik  can you please help to address the comments? Thanks.

File: doc/SONiC feature quality definition.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- For standalone feature it should be considered to have a compilation flag in enabled mode. Feature requires enabled/disabled via config_db and set to disable. 
+- Can go to master branch only
+- For features based on ASIC vendor SAI implementation: SAI is available for at least 1 vendor
+- For feature based on Platform vendor API implementation: available for at least 1 vendor.
+- Should not cause any degradation on production features use relevant sonic-mgmt. tests to confirm
+- New sonic-mgmt. test plan has been reviewed but partially implemented
+
+### GA level
+- Code PRs must come unit tests aligned to code coverage 
+- For standalone feature it should be considered to have a compilation flag in disabled mode. Feature requires enabled/disabled via config_db and set to enabled if HLD claims for it
```
[the remark this answers is not in the document]
Comment (liat-grozovik) [reply doc/SONiC feature quality definition.md:31]: should be removed following the latest discussion in TSC. wil fix

**Comment by liat-grozovik** [comment]
@zhangyanzhao @yxieca the comments were handled. can you please help to approve/merge?

**Review by yxieca — APPROVED** [review]

**Comment by zhangyanzhao** [comment]
community review recording https://zoom.us/rec/share/0iPxMuCMcKlDcqwisDHcTgwGMP5-GddrdRrYmDky9-l_Fvzo_gdALiQZMsxCIDFw.Jvh9HTTlJAjyrFDt

**Pull request merged**